### PR TITLE
Added Loading button , postgres readonly and fixed many UI bugs

### PR DESCRIPTION
**Pull request by dakshgautam**

### CODE REVIEW

File: squealy/views.py [hunk 1]
```diff
@@ -171,12 +171,12 @@ def _run_validations(self, params, user, validations, db):
             run_validation(params, user, validation.query, db)
 
     def _check_read_only_query(self, query):
-        if any(keyword in query.upper() for keyword in SQL_WRITE_BLACKLIST):
-            raise DatabaseWriteException('Database write commands not permitted in the query.')
+        # if any(keyword in query.upper() for keyword in SQL_WRITE_BLACKLIST):
```
Comment (swapnil95) [inline squealy/views.py]: Remove these if not required

File: squealy/views.py [hunk 2]
```diff
@@ -171,12 +171,12 @@ def _run_validations(self, params, user, validations, db):
...
+        #     raise DatabaseWriteException('Database write commands not permitted in the query.')
         pass
 
     def _execute_query(self, params, user, chart_query, db):
-        self._check_read_only_query(chart_query)
+        #self._check_read_only_query(chart_query)
```
Comment (swapnil95) [inline squealy/views.py]: Same here

File: squealy/formatters.py [hunk 3]
```diff
@@ -74,7 +76,7 @@ def format(self, table, chart_type):
 
         for row in table.data:
             for index, data in enumerate(row):
-                if type(data) not in [int, float, long]:
+                if (type(data) not in [int, float, long]) and (isinstance(data, decimal.Decimal) == False):
```
Comment (devashishsharma2302) [inline squealy/formatters.py]: What if we just add decimal.Decimal to the list [int, float, long] ? Will it work?

File: squealy/squealy-web/src/Components/ResultSection.js [hunk 4]
```diff
@@ -6,7 +6,9 @@ import { SquealyDropdown } from './SquealyUtilsComponents'
 import { GOOGLE_CHART_TYPE_OPTIONS } from './../Constant'
 import configIcon from './../images/settings_icon.png'
 import ChartConfigModal from './ChartConfigModal'
-import {ErrorMessagePanel} from './ErrorMessageComponent'
+import { ErrorMessagePanel } from './ErrorMessageComponent'
+import Loading from 'react-loading'
```
Comment (devashishsharma2302) [inline squealy/squealy-web/src/Components/ResultSection.js]: Add this to package.json too!

File: squealy/formatters.py (hunk 3 above)
Comment (dakshgautamm) [inline squealy/formatters.py]: No

**Pull request merged**